service/internal/cloudconfig/template/instance_health_check.go

```diff
@@ -33,7 +33,7 @@ if test $(cut -d '.' -f1 /proc/uptime) -lt 900; then
 fi
 
 # AWS Metadata
-export INSTANCEID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id 2> /dev/null)
+export INSTANCEID=$(/opt/imds-client /latest/meta-data/instance-id)
```

Comment:
just for my curiosity is that client shipped with our k8scc or is flatcar shipping it?

Reply:
k8scc https://github.com/giantswarm/k8scloudconfig/blob/master/files/conf/imds-client